_data/simple-icons.json

```diff
@@ -2300,6 +2300,11 @@
             "hex": "6FDA44",
             "source": "https://www.upwork.com/press/"
         },
+	{
```

Comment:
The indentation is a bit off here - looks like you're using tabs while the rest of the file is using 8 spaces. We'd prefer to keep the indentation consistent throughout.

Reply:
I edited this part following your advise!